### PR TITLE
Fix synchronization of stop/audio buttons with state

### PR DESCRIPTION
**Pull request by alucio-echen**

### CODE REVIEW

File: alucioComponents/components/ParticipantInfo/ParticipantInfo.tsx [hunk 1]
```diff
@@ -142,7 +132,7 @@ export default function ParticipantInfo({ participant, onClick, isSelected, chil
               {getName(participant.identity)}
             </h4>
             <div style={classes.controlsContainer}>
-              <AudioLevelIndicator audioTrack={audioTrack} background="black" />
+              <AudioLevelIndicator audioTrack={audioTrack} />
```
Comment (chibchombiano26) [inline alucioComponents/components/ParticipantInfo/ParticipantInfo.tsx:135]: 👍

File: alucioComponents/components/ParticipantInfo/ParticipantInfo.tsx [hunk 2]
```diff
@@ -154,7 +144,13 @@ export default function ParticipantInfo({ participant, onClick, isSelected, chil
...
       </div>
       {isVideoSwitchedOff && <BandwidthWarning />}
-      {children}
+      <Iffy is={videoTrack?.isEnabled && videoTrack?.isStarted}>
+        {children}
+      </Iffy>
+      <Iffy is={!(videoTrack?.isEnabled && videoTrack?.isStarted)}>
+        <ParticipantPlaceholder/>
+      </Iffy>
+
```
Comment (chibchombiano26) [inline alucioComponents/components/ParticipantInfo/ParticipantInfo.tsx:153]: Could you please remove this empty break line?

File: alucioComponents/components/ParticipantInfo/ParticipantPlaceholder/ParticipantPlaceholder.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const styles = StyleSheet.create({
+  container: {
+    height: '100%',
+    paddingTop: 11,
+    paddingLeft: 88,
+  },
+  icon: {
+    width: 80,
+    height: 80,
+    color: 'rgba(255,255,255,0.4)',
```
Comment (chibchombiano26) [inline alucioComponents/components/ParticipantInfo/ParticipantPlaceholder/ParticipantPlaceholder.tsx:14]: not sure if we can move this to luxcolors

File: alucioComponents/components/Virtual/MeetingControls/MeetingControls.tsx [hunk 4]
```diff
@@ -147,10 +136,23 @@ function MeetingControls(props: HeadBarProps) {
     }
   }
 
-  async function endCall() {
-    props.onCallEnd();
+  if( isAudioEnabled ) {
+    PlayerMenuOptions[0].title = 'Mute'
```
Comment (chibchombiano26) [inline alucioComponents/components/Virtual/MeetingControls/MeetingControls.tsx:140]: probably we need in a further ticket change this array to an object (I think that I introduced this code, my bad), I think would be better something like:
in a hash javascript object.

PlayerMenuOption['Mute']. title = 'Menu'

File: alucioComponents/components/Virtual/MeetingControls/MeetingControls.tsx [hunk 5]
```diff
@@ -167,7 +169,7 @@ function MeetingControls(props: HeadBarProps) {
       </View>
     </View>
     <View style={styles.endCall}>
-      <Button.Kitten style={styles.endCallButton} onPress={endCall} status="danger">
+      <Button.Kitten style={styles.endCallButton} onPress={props.onCallEnd} status="danger">
```
Comment (chibchombiano26) [inline alucioComponents/components/Virtual/MeetingControls/MeetingControls.tsx:172]: 👍

File: alucioComponents/components/Virtual/VirtualExchange.tsx [hunk 6]
```diff
@@ -173,15 +193,33 @@ function Layout(props: LayoutProps) {
     init();
   }, [virtual])
 
+
+
   useEffect(()=>{
-    room.on('participantDisconnected', (e) => {
+    const handleParticipantDisconnected = (e:RemoteParticipant) => {
```
Comment (chibchombiano26) [inline alucioComponents/components/Virtual/VirtualExchange.tsx:199]: 👍

File: alucioComponents/components/ParticipantInfo/ParticipantPlaceholder/ParticipantPlaceholder.tsx (hunk 3 above)
Comment (alucio-echen) [reply alucioComponents/components/ParticipantInfo/ParticipantPlaceholder/ParticipantPlaceholder.tsx:14]: I decided to punt on this for now while we figure out the final naming of colors.

**Pull request merged**